docs/src/main/asciidoc/application-lifecycle-events-guide.adoc

```diff
@@ -94,10 +94,18 @@ TIP: The events are also called in _dev mode_ between each redeployment.
 
 NOTE: The methods can access injected beans. Check the {quickstarts-blob-url}/application-lifecycle-events/src/main/java/org/acme/events/AppLifecycleBean.java[AppLifecycleBean.java] class for details.
 
+=== What is the difference from `@Initialized(ApplicationScoped.class)` and `@Destroyed(ApplicationScoped.class)`
```

Comment:
I would `between` instead of `from` here.

Reply:
-1 That would completely change the meaning... first we introduce `StartupEvent` and `ShutdownEvent` and then in this section we try to explain the difference between `StartupEvent` and `@Initialized(ApplicationScoped.class)` and between `ShutdownEvent` and `@Destroyed(ApplicationScoped.class)`.

Reply:
I see what you're saying, but I don't think it makes grammatical sense to use `from`. 
In any case this is too minor to worry about :)

Reply:
Last point, in the title of the issue  `between` is used instead of `from` :smile:

Reply:
> Last point, ...

Yep, because the title mentions `StartupEvent` and `@Initialized(ApplicationScoped.class)` ;-)

> In any case this is too minor to worry about :)

Ok, I still think that `be different from` is legal in this case. But again I'm no expert ;-).

Reply:
Let's just leave it as it is :)

Reply:
+1 for Martin here :).

Reply:
sold